jaxrs-api/src/main/java/jakarta/ws/rs/LenghtRequiredException.java

```diff
@@ -0,0 +1,106 @@
+/********************************************************************************
+ * Copyright (c) 2019 Jeyvison Nascimento
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ ********************************************************************************/
+
+package jakarta.ws.rs;
+
+import jakarta.ws.rs.core.Response;
+
+/**
+ * A runtime exception indicating the server refuses to accept the request without a defined Content-Length.
+ *
+ * @author Jeyvison Nascimento (jeynoronha@gmail.com)
+ * @since 2.2
+ */
+public class LenghtRequiredException extends ClientErrorException {
```

Comment:
lenght -> length throughout

Reply:
Can you clarify here, @ronsigal ? Didn't understand the comment

Reply:
No need, got it